pyautomagic/src/rateQuality.py

```diff
@@ -0,0 +1,51 @@
+def rateQuality(OHA, THV, CHV, RCB, overallGoodCutoff:float=0.1, overallBadCutoff:float=0.2, timeGoodCutoff:float=0.1, timeBadCutoff:float=0.2,
+                channelGoodCutoff:float=0.15, channelBadCutoff:float=0.3, BadChannelGoodCutoff:float=0.15, BadChannelBadCutoff:float=0.3):
+    
+    # check that thresholds are between 0 and 1
+    if any(isinstance(x, int) for x in [fill_this_in]) or any(x <= 0 or x >= 1 for x in [fill_this_in]):
+           logger.log(f"Some threshold cutoffs were set as integer. Please pass in a float between 0 and 1. You passed in {stuff}."
+           raise ValueError("Say something")
+
+    qualityScores = [OHA, THV, CHV, RCB]
+    Qs = qualityScores
+
+    # Categorize wrt OHA

```

Comment:
Create a separate function:

def something(qs, threshold):
        <the if/else to return good,okay,bad>
       return string